cmd/sink/main.go

```diff
@@ -5,22 +5,151 @@ package main
 
 import (
 	"context"
+	"database/sql"
 	"log"
 	"net/http"
 	"os"
+	"time"
 
 	ceOtelObs "github.com/cloudevents/sdk-go/observability/opentelemetry/v2/client"
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	ceClient "github.com/cloudevents/sdk-go/v2/client"
+	"github.com/cloudevents/sdk-go/v2/types"
 	kaObservability "github.com/kubearchive/kubearchive/pkg/observability"
+	_ "github.com/lib/pq"
 	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
 )
 
-var logger = log.New(os.Stderr, "", log.LstdFlags|log.Lmicroseconds|log.LUTC)
+const (
+	connStr             = "user=kubearchive password=kubearchive dbname=kubearchive-db host=10.96.22.175 port=5432 sslmode=disable"
```

Comment:
To avoid using the IP:

```suggestion
	connStr             = "user=kubearchive password=kubearchive dbname=kubearchive-db host=<db-service-name>.<db-namespace>.svc.cluster.local port=5432 sslmode=disable"
```

Reply:
using the service url for the database now. I'm gonna modify the helm chart so that all of these values get added as env vars to the sink when it is deployed